docs/conceptual/runtime/frame.md

```diff
@@ -1,29 +1,26 @@
 ---
-title: Substrate Runtime Module Library
+title: Framework for Runtime Aggregation of Modularised Entities
```

Comment:
I think this should be "Modularized" with a z like American spelling. [ngram reference](https://books.google.com/ngrams/graph?content=modularize%2Cmodularise&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cmodularize%3B%2Cc0%3B.t1%3B%2Cmodularise%3B%2Cc0)

Double checking with @joepetrowski

Reply:
Up to know we only use `modularised` in many different parts of the docs. 
What will be the correct spelling?
There are some of this lines that point to the glossary. FRAME needs to be added to the glossary.
overview/glossary.md#frame-framework-runtime-aggregation-modularised-entities

Reply:
If ngram has a spread like that, it's the decider :). Use modularize.

I assigned myself the issue for the glossary since I recently updated the Polkadot one. Will get that done over xmas.

Reply:
Thanks. I'll update it to `modularize`